operations/node_management/Configure_NTP_on_NCNs.md

```diff
@@ -77,6 +77,12 @@ The following steps are structured to be executed on one node at a time. However
 
 1. Set a token as described in [Identify Nodes and Update Metadata](Rebuild_NCNs/Identify_Nodes_and_Update_Metadata.md).
```

Comment:
Should this step and the next one be done on each affected NCN? If so, you should put the "On each affected NCN" part above all of these steps.

Reply:
There's two contexts.  The first is running it against every node at once in the `for` loop.  The other is ad-hoc on the node you want.  In that instance, it mentioned the "affected ncn" https://github.com/Cray-HPE/docs-csm/pull/1705/files/8c69cbd5cac23171773edda446ac908539c7aa5f#diff-bdc6d5e32445a6f32b36d568145774ebe9b424e01dded6bb3c1713eabbc676a8R86

Reply:
Okay, but as written the doc says to set the token and then, on all of the affected nodes, run that script. Do you not need to set the token on all of those nodes?

Reply:
I see what you mean I think.  Check out the latest push

Reply:
Yup, that's what I was trying to suggest originally, but I can see how my wording was not ideal.